app/blog/2017-07-12-an-update-on-neon-animation.md

```diff
@@ -0,0 +1,70 @@
+
+Sharing is easy if you restrict your animation parameters to something like keyframes. If the goal is instead to share an animation concept with infinite flexibility, then you’d essentially be exposing the entire animations API and reinventing the wheel.
+
+## Alternatives
+As I've mentioned, animations have not gone away, they're still here in the platform. There are 3 different alternatives that we suggest (in no particular order):
+
+1. CSS Keyframe Animations
+2. Web Animations API
+3. CSS Transitions
+
+### CSS Keyframe Animations
+CSS Keyframe animations are great because they provide great control over animations. Ways they do this: they introduce the concept of a Keyframe which essentially represent a CSS snapshot at a specific percentage of an animation's timeline, CSS Keyframes also provide a nice, declarative syntax to define an animation by stylistically decoupling animations into the CSS. Another nice thing about CSS Keyframes is that they have been [supported](http://caniuse.com/#search=css%20animation) (prefixed) since Obama's first term (unprefixed around when [The Weeknd](https://www.youtube.com/watch?v=KEI4qSrkPAs) stopped feeling their face).
+
+What makes them not so great is trying to make changes to the keyframes; it's possible but it's ugly, painful, and not always efficient. This makes it difficult to make “dynamic” animations that calculate keyframes.
+
+The basics of CSS Keyframe animations are well-covered by [Mozilla](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations).
+
+### Web Animations API
+The [Web Animations API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API) is a great all-around replacement as `neon-animation` was designed around the Web Animations API. It  does everything the CSS Keyframe Animations do, but it also has the luxury of being in Javascript and thus can make changes on the fly. The web animations API is natively supported in Chrome & Firefox, with a [polyfill](https://github.com/web-animations/web-animations-js) required in other browsers [other browsers](http://caniuse.com/#search=web%20animations%20api).
+
+[Mozilla](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Using_the_Web_Animations_API) has a good intro to the API.
+
+### CSS Transitions
+CSS Transitions allow you to animate changes in CSS properties. Transitions can also easily be set via javascript so you don't run into the same issues as CSS Keyframe animations.
+
+The largest limitation of CSS Transitions is that they make it difficult to coordinate animations involving multiple properties or multiple states.
+
+Here's a trusty [MDN link](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions) on how to use them.
```

Comment:
I would love to see (maybe in the summary above, or maybe in a wrap-up below), a quick summary of the advantages and drawbacks to help people choose what to use. For example, you mention CSS transitions making it different to coordinate animations... Are both of the other APIs better at this? How about parallel or serial animations? Does that point you towards Web Animations API?

I would also love to see samples for each of these, but that might make for a really heavy article overall.

Maybe just a single example comparing `neon-animation` config and WA API? After reading this, I feel like I still don't really have a handle on what my next steps are if I'm trying to rewrite a `neon-animation` animation.

Also, be nice to add a wrap up for the whole article at the end reiterating the points from "What should I do."